.github/workflows/main.yaml

```diff
@@ -74,10 +75,50 @@ jobs:
           FLUTTER_DART_HOME: ${{ env.FLUTTER_HOME }}/bin/cache/dart-sdk/bin
         run: |
           export PATH=$PATH:$FLUTTER_DART_HOME
-          ./scripts/ci.sh packages/${{ matrix.package }}
+          ./scripts/ci.sh ${{ matrix.folder }}
 
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v1
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           file: ${{ matrix.package }}.lcov
+
+  # Build an APK for each flutter example.
+  build-apk:
+    runs-on: ${{ matrix.os }}
+    needs: setup
```

Comment:
I think this would need the `analyze-and-test` job more as it's time-consuming to run build when tests or something else fails, as the build probably also fails?

Reply:
Great idea, implemented.